src/KeyUnit.cpp

```diff
@@ -60,14 +60,13 @@ void KeyUnit::uninstall(const QString& packageName)
 
 bool KeyUnit::processDataStream(int key, int modifier)
 {
-    bool isAtLeastOneMatch = false;
     for (auto keyObject : mKeyRootNodeList) {
         if (keyObject->match(key, modifier)) {
-            isAtLeastOneMatch = true;
+            return true;
         }
     }
 
-    return isAtLeastOneMatch;
+    return false;
```

Comment:
As someone who did enter `lua permKey("my key", "", mudlet.keymodifier.Control, mudlet.key.F8, [[echo("hey this thing actually works!\n")]])` several times with different function strings to test it I was surprised when I __didn't__ get several Keys going off - especially as they all showed in the Editor (after refreshing the contents).  

Not sure what the prioritization is but if a script puts up a `tempKey` and I then enter one in the editor with the same key+modifier combination will my (effectively a `permKey`) ever run? IIRC the temporary ones do not show but are given priority (I could easily be rong on that I know) but that must surely be confusing even if it is a corner case... ☹️